cli-tests/README.md

```diff
@@ -0,0 +1,39 @@
+# PMM CLI tests
+Percona Monitoring and Management CLI automated tests. 
+
+
+## Getting Started
+
+* Open the _**cli**_ folder in console
+* Install Node.js 12+ version and make sure npx is included
+* Install project dependencies: `npm ci`
+* Install "playwright": `npx playwright install`
+* Install test module if required: `npm install -D @playwright/test`
+
+this is it! tests are good to go on specified PMM Server and/or Client.
+
+### Getting PMM server and client ready
+  * **Setup PMM Client**
+    * Download and unzip client tarball
+    * Put binaries into `/usr/bin` directory and grant corresponding permissions
```

Comment:
since it's already in the `pmm` repository maybe we can give information on how to build it from sources?